apps/web/composables/useCart/useCart.ts

```diff
@@ -1,6 +1,6 @@
 import type { CartItem } from '@plentymarkets/shop-api';
```

Comment:
here on all imports we have only a few ```type```, rest of them interfaces

Reply:
As you see I dont changed that in this PR.

Reply:
true, true